server/cri/v1/rpc_container_events.go

```diff
@@ -5,6 +5,13 @@ import (
 )
 
 func (s *service) GetContainerEvents(req *pb.GetEventsRequest, ces pb.RuntimeService_GetContainerEventsServer) error {
```

Comment:
a nit about code structure: we have a convention of having functions in this folder this to call directly into a function in the server (see the other rpcs for examples)

Reply:
Addressed